CMakeLists.txt

```diff
@@ -280,9 +280,9 @@ if(FINUFFT_USE_CPU)
 endif()
 
 if(FINUFFT_USE_CUDA)
-  if(NOT DEFINED CMAKE_CUDA_ARCHITECTURES)
```

Comment:
I think, if someone specifies `CMAKE_CUDA_ARCHITECTURES` but not `FINUFFT_CUDA_ARCHITECTURES` we should use `CMAKE_CUDA_ARCHITECTURES`. If none of those is specified then we should default to native.

Reply:
Good point! I have now introduced additional changes to set `FINUFFT_CUDA_ARCHITECTURES` to the value of `CMAKE_CUDA_ARCHITECTURES` if only the latter is defined.